.pullapprove.yml

```diff
@@ -0,0 +1,23 @@
+approve_by_comment: true
+approve_regex: ^Approved
+reject_regex: ^Rejected
+reset_on_push: true
+author_approval: required
+
+reviewers:
+  -
+      name: admins
+      required: 1
+      members:
+        - fred333
```

Comment:
Which, actually, is another example of why handwritten lists of names aren't good.